src/pages/Settings/Settings.tsx

```diff
@@ -412,29 +425,31 @@ const WebauthnCredentialItem = ({
 					)
 					: (
 						<div className="flex items-center">
-							<p className="font-bold">
-								Nickname:&nbsp;	
-							</p>
-							<p className="font-bold text-custom-blue">
-								{currentLabel}
+							<p>
+								<span className="font-semibold">
+									{t('pageSettings.loggedPasskey.nickname')}:&nbsp;	
+								</span>
+								<span className="font-bold text-custom-blue">
+									{currentLabel}
+							</span>
 							</p>
 						</div>
 					)
 				}
 				<p> 
 					<span className="font-semibold">
-						Created:&nbsp;	
+					{t('pageSettings.loggedPasskey.created')}:&nbsp;	
 					</span>
 					{formatDate(credential.createTime)}
 					</p>
 				<p>
 					<span className="font-semibold">
-						Last used:&nbsp;	
+					{t('pageSettings.loggedPasskey.lastUsed')}:&nbsp;	
```

Comment:
Was the indentation change here intentional?

(The one below this should probably have been indented one step further already :slightly_smiling_face:)